Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/CallEvent.hs

```diff
@@ -48,6 +47,8 @@ sendCallDataToKafka :: (EncFlow m r, CacheFlow m r, EsqDBFlow m r, EsqDBReplicaF
 sendCallDataToKafka vendor mRideId callType callSid callStatus triggeredBy exophoneNumber = do
   case mRideId of
     Just rideId -> do
-      ride <- runInReplica $ QRide.findById rideId >>= fromMaybeM (RideDoesNotExist rideId.getId)
-      triggerExophoneEvent $ ExophoneEventData vendor callType (Just rideId) callSid callStatus ride.merchantId triggeredBy (Just ride.driverId) exophoneNumber
+      mRide <- runInReplica $ QRide.findById rideId
+      case mRide of
+        Just ride -> triggerExophoneEvent $ ExophoneEventData vendor callType (Just rideId) callSid callStatus ride.merchantId triggeredBy (Just ride.driverId) exophoneNumber
+        Nothing -> triggerExophoneEvent $ ExophoneEventData vendor callType (Just rideId) callSid callStatus Nothing triggeredBy Nothing exophoneNumber
```

Comment:
Is it  suppose to be like this ?
Don't you need to mention that it is RC related call ?

Reply:
We are checking if the callService is active or not using this right? won't it same for RC calls too?